stackcopy.py

```
@@ -14,10 +13,12 @@
 import argparse
 import re
 import errno
+from __future__ import annotations
```

Ruff (0.14.13)

`from __future__` imports must occur at the beginning of the file

(F404)